pkg/codegen/jenny_crd_watcher.go

```diff
@@ -31,12 +31,16 @@ func (j crdWatcherJenny) Generate(kind kindsys.Kind) (*codejen.File, error) {
 		return nil, nil
 	}
 
+	name := kind.Props().Common().MachineName
+	if name == "dashboard" {
```

Comment:
HACK to avoid typed watcher

Reply:
looks like there's also a typo on line 29, should be kindsys.Custom no?

Reply:
good catch @DanCech

Reply:
updated